### DISCUSSION TITLE
Docker install: Can delete entries in demo but not add new notes anywhere

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [TriliumNext/Trilium#4715] [Q&A]
I am obviously doing something wrong but when I click on "New Note", nothing happens. I can however delete existing notes.

I see that I am in a protected session but clicking on the shield does nothing.

**Comment by ZuluPilgrim** [comment]
I suddenly have a heap of new notes.. looks like there was a delay. I might need to add more memory or something to my system

**Reply by zadam** [reply]
That's strange. Problems like this are often caused by blocked WebSocket connection in the HTTP proxy.
